### DISCUSSION TITLE
feature: Export of current chat

### DISCUSSION THREAD

**Discussion by nksCH** [bionic-gpt/bionic-gpt#334] [Announcements]
### Describe your issue

I cannot find a button in the UI to export the current chat.
This might be a useful feature.

### Describe your setup/startup scripts

N/A

### Steps to reproduce

1. Go to chat
2. Converse with the model
3. No button to export

**Comment by ianpurton** [comment]
What format would expect the export to be in?

**Reply by nksCH** [reply]
The export should be either JSON or preferably Markdown.
